src/Compiler/OnnxMlirCompiler.cpp

```diff
@@ -7,14 +7,47 @@
 
 extern "C" {
 namespace onnx_mlir {
-ONNX_MLIR_EXPORT int omCompileFromFile(const char *inputFilename,
+
+ONNX_MLIR_EXPORT int64_t omSetCompilerOptionsFromEnv(const char *envVarName) {
+  // ParseCommandLineOptions needs at least one argument
+  std::string nameStr = "program-name";
+  const char *argv[1];
+  argv[0] = nameStr.c_str();
+  const char *name = envVarName ? envVarName : OnnxMlirEnvOptionName.c_str();
+  return llvm::cl::ParseCommandLineOptions(
```

Comment:
ParseCommandLineOptions returns true on success, but we should return 0 on success.

Reply:
Thanks, good catch!